content/lib/2021-04-02-operating-system-interface.md

```diff
@@ -0,0 +1,225 @@
+---
```

Comment:
please suggest links, references, blogs, tutorials, etc that will validate or invalidate any of this

even if it's just a thought that it might "have something to do with" please share